src/web/mjs/connectors/Jmana1.mjs

```diff
@@ -0,0 +1,71 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class Jmana1 extends Connector {
+    constructor() {
+        super();
+        super.id = 'jmana1';
+        super.label = '제이마나 {Jaymana)';
+        this.tags = [ 'manga', 'korean' ];
+        this.url = 'https://jmana1.net';
+
+        this.path = '/comic_main';
+        this.queryMangas = 'div.section div div.conts ul.allList li a';
+        this.queryChapters = 'div.post-content-list h2 a';
+        this.queryPages = 'div ul.listType.view li#view_content2 source';
+    }
+
+    async _getMangas() {
+        let mangaList = [];
+        for(let page = 1, run = true; run; page++) {
+            let mangas = await this._getMangasFromPage(page);
+            mangas.length ? mangaList.push(...mangas) : run = false;
+        }
+        return mangaList;
+    }
+
+    async _getMangasFromPage(page) {
+        // NOTE: Avoid 429 Too Many Request
+        await new Promise(r => setTimeout(r, 2000));
+        const uri = new URL(this.path, this.url);
+        uri.searchParams.set('page', page);
+        const request = new Request(uri, this.requestOptions);
+        const data = await this.fetchDOM(request, this.queryMangas);
+        return data.map(element => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element, this.url),
+                title: element.querySelector('span.titBox span.price').textContent.trim()
```

Comment:
The nam e os the CSS class `.price` is a little bit suspicious.
Does this work with free, non-purchased and also purchased content?

Reply:
This is just how the site structure works for all titles